open_graph.py

```diff
@@ -50,7 +55,9 @@ def tag_article(instance):
     ogtags.append(('og:site_name', instance.settings.get('SITENAME', '')))
 
     ogtags.append(('article:published_time', strftime(instance.date, "%Y-%m-%d")))
-    ogtags.append(('article:modified_time', strftime(instance.modified, "%Y-%m-%d")))
+    
+    if hasattr(instance, 'modified'):
+        ogtags.append(('article:modified_time', strftime(instance.modified, "%Y-%m-%d")))
 
```

Comment:
Please restrict your PR to one issue. This change is already the object of another PR.